### PR TITLE
fix: update probability percentage validation

### PR DESCRIPTION
**Pull request by davehlewis**

### CODE REVIEW

File: gef-ui/server/controllers/about-exporter/index.js [hunk 1]
```diff
@@ -78,12 +78,12 @@ const validateAboutExporter = async (req, res) => {
       }
     }
 
-    // Regex tests to see if value is a number between 0 and 100
-    const zeroToOneHundredRegex = /^([0-9]|[1-9][0-9]|100)$/;
-    if (body.probabilityOfDefault && !zeroToOneHundredRegex.test(body.probabilityOfDefault)) {
+    const maxPercentage = 14.1;
+    const probabilityOfDefault = Number(body.probabilityOfDefault);
+    if (Number.isNaN(probabilityOfDefault) || probabilityOfDefault > maxPercentage) {
```
Comment (pbradley-f4) [inline gef-ui/server/controllers/about-exporter/index.js:83]: Not one for the first day but extract this to a utils method of isNumberGreaterThan(maxPercentage) or the like

**Review by pricea1 — COMMENTED** [review]
Agree with Pauls comments otherwise code lgtm.

In future, can you add the ticket reference at the start of the commit message
e.g. dtfs2-3969 update probability

**Review by pricea1 — APPROVED** [review]

File: gef-ui/server/controllers/about-exporter/index.js (hunk 1 above)
Comment (davehlewis) [reply gef-ui/server/controllers/about-exporter/index.js:83]: This is a specific use case for validation so doesn't really suit being a util. Returns true if it is not a number for example.

File: gef-ui/server/controllers/about-exporter/index.js [hunk 2]
```diff
@@ -78,12 +78,17 @@ const validateAboutExporter = async (req, res) => {
...
+    const probabilityOfDefault = Number(body.probabilityOfDefault);
+    if (
+      Number.isNaN(probabilityOfDefault)
+      || !precisionTwoRegex.test(body.probabilityOfDefault)
+      || probabilityOfDefault > maxPercentage
+    ) {
       aboutExporterErrors.push({
         errRef: 'probabilityOfDefault',
-        errMsg: 'You must enter a percentage between 0% to 100%',
+        errMsg: `You must enter a percentage between 0% to ${maxPercentage}}%`,
```
Comment (pbradley-f4) [inline gef-ui/server/controllers/about-exporter/index.js:91]: too many `}`

Comment (davehlewis) [reply gef-ui/server/controllers/about-exporter/index.js]: Sorry should have caught that

**Pull request merged**